### DISCUSSION TITLE
Support volume adjustment for  AUX IN sound bar connected to monitor

### DISCUSSION THREAD

**Discussion by CHENG-LIANG1** [MonitorControl/MonitorControl#1274] [General]
### Before opening the issue, have you...?

- [X] Searched for existing issues

### Is your feature request related to a problem? Please describe

1. Connect a sound bar to a monitor via AUX IN cable
2. Adjust volume
3. Nothing happens

### Describe the solution you'd like

1. Connect a sound bar to a monitor via AUX IN cable
2. Adjust volume
3. Volume changes

### Describe alternatives you've considered

1. Use the app for sound bar to adjust volume
2. Use physical buttons

**Comment by waydabber** [comment]
Sadly a lot of displays do not properly support DDC volume controls or might need special, highly manufacturer specific commands to switch controllable audio output. You can of course try experimenting with your specific hardware by altering the app's code and see if there is a solution to which the display responds.
